pages/entry.jsx

```diff
@@ -5,7 +5,8 @@ import ProjectCard from '../components/project-card/project-card.jsx';
 import Service from '../js/service.js';
 import Utility from '../js/utility.js';
 
-const NO_ENTRY_TITLE = `Entry unavailable`;
+const NO_ENTRY_OG_TITLE = `Entry unavailable`;
```

Comment:
This has to stay `NO_ENTRY_TITLE` because this is data used for the page. We set the DOM's `<title>` element to this value. As such, rather than renaming it, keep it the old name, and instead just use that same value in the `meta property="og:title"` part (which you're already doing due to using `docTitle`). We don't need to put the "OG" part in the variable name, think of it instead as  putting "normal data" into opengraph meta tags as well. None of this data is strictly speaking "reserved for use by opengraph only".